src/components/BurnerWalletCard/BurnerWalletHeaderLogo.tsx

```diff
@@ -1,17 +1,20 @@
 import { displayFrom, displayTo } from 'helpers/visibilityClassnames'
 import BurnerWalletLogo from 'icons/BurnerWalletLogo'
-import classnames, { maxWidth } from 'classnames/tailwind'
-
-const smallLogo = classnames(maxWidth('max-w-84'), displayTo('md'))
+import BurnerWalletSeal from 'icons/BurnerWalletSeal'
 
 export default function () {
   return (
     <>
       <div className={displayFrom('md')}>
         <BurnerWalletLogo />
       </div>
-      <div className={smallLogo}>
-        <BurnerWalletLogo small />
+      <div className={displayTo('md')}>
```

Comment:
Wait, so the whole block of `wallets` will be displayed only before `md`?

Reply:
No, only Logo part